addons/mail/static/src/core/common/attachment_model.js

```diff
@@ -47,6 +47,9 @@ export class Attachment extends Record {
         if (!("extension" in data) && data["name"]) {
             this.extension = this.name.split(".").pop();
         }
+        if ("access_token" in data) {
+            this.accessToken = data.access_token;
+        }
```

Comment:
Keep same name as in python formatter, so `access_token`.
I don't want to write many LOCs of name conversion just to respect the convention in JS to camelCase and snake_case in python. I think this is stupid to accept that when the cost is more LOCs and less traceability in code, prone to far more programming errors and code maintenance than necessary.

Also I have a PR that will soon be merged that introduce `assignIn` helper.
This allows to define many `if(x in data) this.x = data.x` in just a single like of code:
```js
assignIn(this, data, ['x']);
```